src/compiler/types/tests/generate-app-types.spec.ts

```diff
@@ -1406,6 +1407,198 @@ declare module "@stencil/core" {
                 }
         }
 }
+`,
+      {
+        immediateWrite: true,
+      }
+    );
+  });
+
+  it('should transform aliased paths if the config option is true', async () => {
+    // TODO(STENCIL-223): remove spy to test actual resolution behavior
+    jest.spyOn(ts, 'resolveModuleName').mockReturnValue({
+      resolvedModule: {
+        isExternalLibraryImport: false,
+        extension: Extension.Dts,
+        resolvedFileName: 'utils',
+      },
+    });
```

Comment:
so for now the test isn't testing that the values in `paths` are used - I assume this was the stuff you had trouble with testing? when testing the `rewrite-aliased-paths` transformer I was able to get this to work by making sure I called `patchTypescript`, which then let me create files in the in-memory file system (see this setup function for instance: https://github.com/ionic-team/stencil/blob/39ab8d9c00be08216d82ce984bbf7a28c28a93f2/src/compiler/transformers/test/rewrite-aliased-paths.spec.ts#L21-L51)

Any chance a similar approach would work here?

Reply:
Ooh nifty! I'll check this out today.